src/luascript.cpp

```diff
@@ -12608,6 +13360,383 @@ int LuaScriptInterface::luaPartySetSharedExperience(lua_State* L)
 	return 1;
 }
 
+// Loot
+
+int LuaScriptInterface::luaCreateLoot(lua_State* L)
+{
+	// Loot() will create a new loot item
+	Loot* loot = new Loot();
```

Comment:
It's acutally intended so we can make use to add them to different monsters easily without re creating the same loot item again.
ex:
goldCoin = Loot()
goldCoin:setId("gold coin")
goldCoin:setMaxCount(99)

monster = MonsterType("demon")
monster:addLoot(goldCoin)

monster = MonsterType("troll")
goldCoin:setMaxCount(20)
monster:addLoot(goldCoin)


I guess you get my idea, I could probably delete it on addLoot / addChildLoot but then again my idea wouldn't work.